upstream-community-operators/crossplane/0.9.0/crossplane.v0.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: crossplane.v0.9.0
+  namespace: placeholder
+  annotations:
+    capabilities: "Basic Install"
+    categories: "OpenShift Optional"
```

Comment:
"Other" may be the best [category](https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#categories) for now.